mopidy_musicbox_webclient/static/js/gui.js

```diff
@@ -316,7 +320,8 @@ function initSocketevents () {
     })
 
     mopidy.on('event:streamTitleChanged', function (data) {
-        setSongTitle(data.title, true)
+        // Update all track info.
+        mopidy.playback.getCurrentTlTrack().then(processCurrenttrack, console.error)
```

Comment:
Not a bug fix, but I think it is safer to run through all of the ``processCurrenttrack`` logic when the stream title changes (which also updates the UI and all of the popup menus).